### PR TITLE
feat: add config db query check

### PR DESCRIPTION
**Pull request by Kaitou786**

### CODE REVIEW

File: checks/config_db.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	result := pkg.Success(check, ctx.Canary)
+	var results pkg.Results
+	results = append(results, result)
+	logger.Infof("Sending GET request to: %v with query param: %v", check.Host, check.Query)
+	var url string
+	if strings.HasSuffix(check.Host, "/") {
+		url = fmt.Sprintf("%v?query=%v", check.Host, check.Query)
+	} else {
+		url = fmt.Sprintf("%v/?query=%v", check.Host, check.Query)
+	}
```
Comment (Kaitou786) [inline checks/config_db.go:41]: @moshloop  let me know if the endpoint needs to be updated

File: checks/config_db.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	return results
+}
+
+func (c *ConfigdbChecker) Check(ctx *canaryContext.Context, extConfig external.Check) pkg.Results {
+	check := extConfig.(v1.ConfigDBCheck)
+	result := pkg.Success(check, ctx.Canary)
+	var results pkg.Results
+	results = append(results, result)
+	logger.Infof("Sending GET request to: %v with query param: %v", check.Host, check.Query)
```
Comment (moshloop) [inline checks/config_db.go:36]:
```suggestion
	logger.Tracef("[%v] query: %v", check.Host, check.Query)
```

File: checks/config_db.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	var url string
+	if strings.HasSuffix(check.Host, "/") {
+		url = fmt.Sprintf("%v?query=%v", check.Host, check.Query)
```
Comment (moshloop) [inline checks/config_db.go]: We might need to url encode the query

Comment (Kaitou786) [reply checks/config_db.go]: Added the QueryEscape parameter for query

File: checks/config_db.go [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusAccepted && resp.StatusCode != http.StatusCreated {
+		return results.Failf("Failed to get response, %v", resp.Status)
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return results.Failf("Failed to read the response body: %v", err)
+	}
+
+	result.AddDetails(body)
```
Comment (moshloop) [inline checks/config_db.go:57]: Can we unmarshal the json into https://github.com/flanksource/config-db/blob/main/api/v1/interface.go#L41 ?

We can copy and pate the struct definition.

**Pull request merged**